_tutorials/102_march/step-09/102_march_step-09-desc.html

```diff
@@ -1,9 +1,11 @@
 <div>
-    <p id="tutorialGoal">You should be prepared now to encode the note values of the final two measures of the example.</p>
+    <p id="tutorialGoal">Step nine: You should now be ready to encode the note values of the final two measures of the example.</p>
```

Comment:
Following your pattern of the other steps, this could be: 

`Step nine: encoding the note values of ...`

Reply:
Ups. Done.